uw-frame-components/staticFeeds/notifications.json

```diff
@@ -1,5 +1,17 @@
 {"notifications" :
     [
-      
+      {
+        "id"     : 1,
+        "groups" : ["Everyone"],
+        "title"  : "This is an admin notification smoke test.",
+        "actionURL" : "http://www.google.com",
+        "actionAlt" : "Google",
+        "dismissable" : false,
+        "priority" : true,
+		"buttonText" : [
+		  {"caption" : "Feedback", "url" : "/exclusive/feedback"},
+		  {"caption" : "Return To Old", "url" : "my.wisconsin.edu"}
+		]
+      }
```

Comment:
I was going off of Doug's commits, where he moved his changes from sample into this file. I can move them back into sample if that's where they should be. I'm guessing it was moved here in preparation for [MUMUP-2765](https://jira.doit.wisc.edu/jira/browse/MUMMNG-2765). Is that right, @Doug-Reed?